cmake/kokkos_enable_options.cmake

```
@@ -103,6 +103,8 @@ mark_as_advanced(Kokkos_ENABLE_IMPL_MDSPAN)
 mark_as_advanced(Kokkos_ENABLE_MDSPAN_EXTERNAL)
 mark_as_advanced(Kokkos_ENABLE_IMPL_SKIP_COMPILER_MDSPAN)
 
+kokkos_enable_option(NO_EXCEPTIONS ON BOOL "Whether to ban the use of exceptions")
```

The current wording doesn't make it clear to me that the scope is limited to `Kokkos` itself and we wouldn't just add `-fno-excpetions` to the compiler flags.
```suggestion
kokkos_enable_option(NO_EXCEPTIONS ON BOOL "Whether to allow Kokkos to use exceptions")
```